spec/lib/open_food_network/order_cycle_form_applicator_spec.rb

```diff
@@ -27,7 +27,7 @@ module OpenFoodNetwork
         expect(applicator).to receive(:exchange_exists?).with(supplier_id, coordinator_id,
                                                               true).and_return(false)
         expect(applicator).to receive(:add_exchange).with(supplier_id, coordinator_id, true,
-                                                          variant_ids: [1, 3], enterprise_fee_ids: [1, 2], receival_instructions: 'receival instructions')
+                                                          variant_ids: [1, 3], enterprise_fee_ids: [1, 2], receival_instructions: 'receival instructions', tag_list: 'wholesale')
```

Comment:
`tag_list` is supposed to be an array, no?

Reply:
`tag_list` should be a string and `tags` should be an array:

```json
{
  "tags": [
    {
      "text": "incoming"
    },
    {
      "text": "incoming2"
    },
    {
      "text": "coucou"
    }
  ],
  "tag_list": "incoming,incoming2,coucou",
}
```